### ISSUE TITLE
Xcode 7 UI Testing - Cannot access drawer

### ISSUE THREAD

**Issue by moritzdietsche** [mutualmobile/MMDrawerController#424]
When recording a new test case with Xcode 7, elements inside the drawer are not visible to Xcode / Accessibility. Manually navigating the tree that is visible to Accessibility in code, I cannot find my content anywhere. Has anyone tried this before or is this a bug in Xcode? I'm new to UI Testing in Xcode so any advice is welcome.

**Comment by rojo69** [comment]
Same here.
Feels like a bug in Xcode.
